src/deadline/client/cli/_groups/attachment_group.py

```diff
@@ -35,23 +36,24 @@ def cli_attachment():
     help="BETA",
 )
 @click.option(
+    "-m",
     "--manifests",
     multiple=True,
     required=True,
     help="Comma separated file paths to manifest formatted files.",
 )
 @click.option(
-    "--s3-root-path", help="Job Attachments S3 root path including bucket name and root prefix."
+    "--s3-root-uri", help="Job Attachments S3 root uri including bucket name and root prefix."
```

Comment:
nice! I like this, uri is a good type to use here. It can be generic like File or S3